docs/data/material/getting-started/templates/sign-in-side/ToggleColorMode.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import IconButton from '@mui/material/IconButton';
+
+import ModeNightRoundedIcon from '@mui/icons-material/ModeNightRounded';
+import WbSunnyRoundedIcon from '@mui/icons-material/WbSunnyRounded';
+
+function ToggleColorMode({ mode, toggleColorMode, sx }) {
+  return (
+    <IconButton
+      onClick={toggleColorMode}
+      color="primary"
+      aria-label="Theme toggle button"
+      sx={sx}
```

Comment:
We don't seem to need the `sx` prop here in any of these ToggleColorMode components